python/examples/generate_payload_and_schedule_and_apply_latter_to_former.py

```diff
@@ -0,0 +1,93 @@
+import tempfile
+import subprocess
+
+from mlir import ir
+from mlir.dialects import arith, func, linalg, tensor, transform
+from mlir.dialects.transform import structured
+
+from lighthouse import transform as lh_transform
+
+
+def example_payload() -> ir.Module:
+    payload = ir.Module.create()
+    with ir.InsertionPoint(payload.body):
```

Comment:
This should also be a helper, eventually be part of the MLIR importer module, which should be reused by the schedule, too, if we all agree with this design: https://github.com/llvm/lighthouse/wiki/Integrator

Reply:
What would the helper do here? Wrap a payload in a module?